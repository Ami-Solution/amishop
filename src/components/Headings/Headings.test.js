/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { H1, H2, H3 } from './index';

describe('Components Headings', () => {
  it('should render H1', () => {
    const component = renderer.create(<H1>H1</H1>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a light H1', () => {
    const component = renderer.create(<H1 light>H1</H1>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a light H2', () => {
    const component = renderer.create(<H2 light>H2</H2>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a light H2', () => {
    const component = renderer.create(<H2 light>H2</H2>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a light H3', () => {
    const component = renderer.create(<H3 light>H3</H3>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a light H3', () => {
    const component = renderer.create(<H3 light>H3</H3>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
