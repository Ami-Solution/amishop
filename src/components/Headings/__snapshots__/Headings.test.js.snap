// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Headings should render H1 1`] = `
.c0 {
  font-weight: 600;
  font-size: 2rem;
}

<h1
  className="c0"
>
  H1
</h1>
`;

exports[`Components Headings should render a light H1 1`] = `
.c0 {
  font-weight: 600;
  font-weight: 200;
  font-size: 2rem;
}

<h1
  className="c0"
>
  H1
</h1>
`;

exports[`Components Headings should render a light H2 1`] = `
.c0 {
  font-weight: 600;
  font-weight: 200;
  font-size: 1.8rem;
}

<h2
  className="c0"
>
  H2
</h2>
`;

exports[`Components Headings should render a light H2 2`] = `
.c0 {
  font-weight: 600;
  font-weight: 200;
  font-size: 1.8rem;
}

<h2
  className="c0"
>
  H2
</h2>
`;

exports[`Components Headings should render a light H3 1`] = `
.c0 {
  font-weight: 600;
  font-weight: 200;
  font-size: 1.6rem;
}

<h3
  className="c0"
>
  H3
</h3>
`;

exports[`Components Headings should render a light H3 2`] = `
.c0 {
  font-weight: 600;
  font-weight: 200;
  font-size: 1.6rem;
}

<h3
  className="c0"
>
  H3
</h3>
`;
